translations.yml

```diff
@@ -1090,33 +1097,41 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  ja: '編集者'
```

Comment:
@hiroya-uga I see you have sometimes added a colon (see for example your translation of "Status:"), sometimes not (here for example). Is there a particular reason? How are colons used in Japanese?

Reply:
@remibetin Sorry, I made a mistake! It needs a colon!
Diff: https://github.com/w3c/wai-website-data/compare/c471a505063506d9bcd988be9781fe467320bd93..a30665ad8a54cc00466e240a3e836178526bf1d3